core/dungeon-masters-guide/items/items-weapons.xml

```diff
@@ -538,6 +538,211 @@
+			<p class="indent">Despite its nihilism, <em>Blackrazor</em> feels a strange kinship to <em>Wave</em> and <em>Whelm</em>, two other weapons locked away under White Plume Mountain. It wants the three weapons to be united again and wielded together in combat, even though it violently disagrees with <em>Whelm</em> and finds <em>Wave</em> tedious.</p> 
+			<p class="indent"><em>Blackrazor's</em> hunger for souls must be regularly fed. If the sword goes three days or more without consuming a soul, a conflict between it and its wielder occurs at the next sunset.</p>
+		</description>
+		<setters>
+			<set name="keywords">greatsword</set>
+			<set name="category">Magic Weapons</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="greatsword">Weapon</set>
+			<set name="attunement" addition="by a creature of non-lawful alignment"> true</set>
+			<set name="rarity">Legendary</set>
+			<set name="enhancement">3</set>
+			<set name="weapon">Greatsword</set>
+		</setters>
+	</element>
+	<element name="Axe of the Dwarvish Lords" type="Magic Item" source="Dungeon Master’s Guide" id="ID_WOTC_DMG_MAGIC_ITEM_AXE_OF_THE_DWARVISH_LORDS">
+		<description>
+			<p>Seeing the peril his people faced, a young dwarf prince came to believe that his people needed something to unite them. Thus, he set out to forge a weapon that would be such a symbol.</p>
+			<p class="indent">Venturing deep under the mountains, deeper than any dwarf had ever delved, the young prince came to the blazing heart of a great volcano. With the aid of Moradin, the dwarven god of creation, he first crafted four great tools: the <em>Brutal Pick</em>, the <em>Earthheart Forge</em>, the <em>Anvil of Songs</em>, and the <em>Shaping Hammer</em>. With them, he forged the <em>Axe of the Dwarvish Lords</em>.</p>
+			<p class="indent">Armed with the artifact, the prince returned to the dwarf clans and brought peace. His axe ended grudges and answered slights. The clans became allies, and they threw back their enemies and enjoyed an era of prosperity. This young dwarf is remembered as the First King. When he became old, he passed the weapon, which had become his badge of office, to his heir. The rightful inheritors passed the axe on for many generations.</p>
+			<p class="indent">Later, in a dark era marked by treachery and wickedness, the axe was lost in a bloody civil war fomented by greed for its power and the status it bestowed. Centuries later, the dwarves still search for the axe, and many adventurers have made careers of chasing after rumors and plundering old vaults to find it.</p>
+			<p class="indent"><i><b>Magic Weapon.</b></i> The <em>Axe of the Dwarvish Lords</em> is a magic weapon that grants a +3 bonus to attack and damage rolls made with it. The axe also functions as a <em>belt of dwarvenkind</em>, a <em>dwarven thrower</em>, and a <em>sword of sharpness</em>.</p>
```

Comment:
Missing all non dwarf effects from the belt of dwarvenkind

Reply:
lines 703 - 706 are the non-dwarf effects